pages/signin.tsx

```diff
@@ -38,49 +72,94 @@ const SignIn: NextPage = () => {
+								<label htmlFor='password' className='text-sm text-gray-600'>
+									password
+								</label>
 								<input
 									type='password'
+									id='password'
 									required
-									className='input-field outline-none'
+									className='relative flex w-full appearance-none justify-center rounded-md border border-gray-300 px-3 py-2 text-gray-900 placeholder-gray-500 focus:z-10 focus:border-indigo-500 focus:outline-none focus:ring-indigo-500 sm:text-sm'
 									placeholder='*****'
 									value={password}
 									onChange={e => setPassword(e.target.value)}
 								/>
 							</div>
+							{/*TODO: REMEMBER ME AND FORGOT PASSWORD FUNCTIONALITY */}
+							{/* <div className='flex items-center justify-between'>
+								<div className='flex items-center'>
+									<input
+										id='remember-me'
+										name='remember-me'
+										type='checkbox'
+										className='h-4 w-4 rounded border-gray-300 text-indigo-600 focus:ring-indigo-500'
+									/>
+									<label htmlFor='remember-me' className='ml-2 block text-sm text-gray-900'>
+										{' '}
+										Remember me{' '}
+									</label>
+								</div>
+
+								<div className='text-sm'>
+									<a href='#' className='font-medium text-indigo-600 hover:text-indigo-500'>
+										{' '}
+										Forgot your password?{' '}
+									</a>
+								</div>
+							</div> */}
+							<button
+								type='submit'
+								className='group relative my-4 flex w-full justify-center rounded-md border border-transparent bg-indigo-600 py-2 px-4 text-sm font-medium text-white hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2'
+							>
+								<span className='absolute inset-y-0 left-0 flex items-center pl-3'>
+									<svg
+										className='h-5 w-5 text-indigo-500 group-hover:text-indigo-400'
+										xmlns='http://www.w3.org/2000/svg'
+										viewBox='0 0 20 20'
+										fill='currentColor'
+										aria-hidden='true'
+									>
+										<path
+											fillRule='evenodd'
+											d='M5 9V7a5 5 0 0110 0v2a2 2 0 012 2v5a2 2 0 01-2 2H5a2 2 0 01-2-2v-5a2 2 0 012-2zm8-2v2H7V7a3 3 0 016 0z'
+											clipRule='evenodd'
+										/>
+									</svg>
+								</span>
+								Sign in
+							</button>
 						</form>
-						<button
-							type='submit'
-							className='my-4 flex  w-[90%] cursor-pointer items-center  justify-center rounded-md bg-blue-600 py-2 font-semibold text-white hover:bg-blue-500'
-							onClick={handleSignIn}
-						>
-							Login
-						</button>
-						<p className='font-semibold'>
-							Not registered yet?
+
+						<p className='my-2 flex w-full justify-center text-sm text-gray-700'>
+							Not registered yet?{' '}
 							<Link className='cursor-pointer font-bold' href={'/signup'}>
-								<a className=' text-blue-600 hover:underline'> Sign up</a>
+								<a className='mx-1 text-blue-600 hover:underline'> Sign up</a>
```

Comment:
No need to add space here as you already added a `{' '}` on line 149 after `Not registered yet?`
```suggestion
								<a className='mx-1 text-blue-600 hover:underline'>Sign up</a>
```